### ISSUE TITLE
Feature: Bind article series/pages

### ISSUE THREAD

**Issue by TheOneValen** [wallabag/wallabag#2430]
I have an Idea for a feature that I would find very useful.

I read a lot of (blog) articles which are divided into multiple pages (without "print" option trick) or a series of articles ("Blabla Part 1 of 3").

It would be cool to have some entity to group articles that belong together. Like a "binding".

Have a nice Day,
Valen

**Comment by nicosomb** [comment]
Hello @TheOneValen! 

This feature is already implemented and must be configured for each website with pagination.

**Comment by j0k3r** [comment]
Well, it's different from a pagination.
In that case, blogger just post 3 differents articles without a pagination. It's 3 differents articles.
I think you miss reading the issue :)

**Comment by nicosomb** [comment]
> I think you miss reading the issue :)

The issue? Which issue? :trollface: 

Sorry, it's too early for me.

**Comment by TheOneValen** [comment]
> Well, it's different from a pagination.
In that case, blogger just post 3 differents articles without a pagination. It's 3 differents articles.

Exactly!

Though @nicosomb how can I handle paginated articles, I didn't find anything in the docs.

**Comment by tcitworld** [comment]
It uses `next_page_link` in the site config : http://help.fivefilters.org/customer/en/portal/articles/223153-site-patterns

**Comment by TheOneValen** [comment]
Ah! Cool. I just tried it for a site and it just worked! I'm impressed.

So I would suggest this feature for multi article series only.

**Comment by tcitworld** [comment]
> So I would suggest this feature for multi article series only.

As of now I suggest adding the same tag to all of the articles.

**Comment by TheOneValen** [comment]
> As of now I suggest adding the same tag to all of the articles.

Yes, thats my workaround.

Sorry I seem to have edited my post while you posted.

**Comment by tcitworld** [comment]
> So I would suggest this feature for multi article series only.

Well, it's doable only if we have a way to link the multiple article between them.

**Comment by TheOneValen** [comment]
> Well, it's doable only if we have a way to link the multiple article between them.

Often bloggers tend to link the previous articles of a series, but that is not very reliable.

Maybe using the tags system, another "type" of tag which is not displayed in the tags overview but in a "series" overview or something like that?

**Comment by tcitworld** [comment]
> Maybe using the tags system, another "type" of tag which is not displayed in the tags overview but in a "series" overview or something like that?

It can be. But it would have to be made manually by the user I guess.

**Comment by TheOneValen** [comment]
Yes. If one expects many more articles in this series, one could use the rule system.

**Comment by TheOneValen** [comment]
Implementing an "abstraction" over Tags also would make it easy to make a category system. I use many tags to categorize articles, but it would be nice to have some "strong" tags (like categories) for recipes for example.
